config.ini

```diff
@@ -0,0 +1,8 @@
+[Hosts]
+FundStore = https://funding-service-design-fund-store-dev.london.cloudapps.digital
+ApplicationStore = https://funding-service-design-application-store-dev.london.cloudapps.digital
+RoundStore = https://funding-service-design-round-store-dev.london.cloudapps.digital
+
+[tool.black]
```

Comment:
create a pyproject.toml and add this to there. Otherwise black may not pick it up.